sos/report/plugins/ovn_host.py

```diff
@@ -75,3 +76,20 @@ def setup(self):
 class DebianOVNHost(OVNHost, DebianPlugin, UbuntuPlugin):
 
     packages = ('ovn-host', )
+    pid_paths = [
+        '/var/snap/openstack-hypervisor/common/run/ovn',
+    ]
+
+    def setup(self):
+
+        if self.is_installed('openstack-hypervisor'):
+            self.ovs_cmd_pre = "openstack-hypervisor."
+
+            common_path = "/var/snap/openstack-hypervisor/common"
```

Comment:
Why not directly `ovn_log_path = "/var/snap/openstack-hypervisor/common/var/log/ovn"`? Or you will use `common_path` else-where, right?